integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseConversionPreviewViewModel.kt

```diff
@@ -162,20 +162,17 @@ class CoinbaseConversionPreviewViewModel @Inject constructor(
     }
 
 
-    private fun sellDashToCoinBase(coin: Coin) = viewModelScope.launch(Dispatchers.Main) {
+    private suspend fun sellDashToCoinBase(coin: Coin) {
```

Comment:
If this method is only called from a coroutine context, why not make it a coroutine? Better for testing and error handling.
`viewModelScope.launch` or other coroutine context launchers should be used as little as possible.